src/styles/custom-antd.less

```diff
@@ -57,22 +57,22 @@
 // @progress-circle-text-font-size: 1em;
 
 // Slider
```

Comment:
I forgot to mention that you don't need to fill all the commented variables in, only the ones you need :)